velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -36,22 +36,25 @@ struct TestParam {
   TypeKind comparisonType;
 };
 
-const std::unordered_set<TypeKind> kSupportedTypes = {
-    TypeKind::BOOLEAN,
-    TypeKind::TINYINT,
-    TypeKind::SMALLINT,
-    TypeKind::INTEGER,
-    TypeKind::BIGINT,
-    TypeKind::REAL,
-    TypeKind::DOUBLE,
-    TypeKind::VARCHAR,
-    TypeKind::TIMESTAMP};
+const std::vector<std::string> kColumnNames =
+    {"c0", "c1", "c2", "c3", "c4", "c5", "c6", "c7", "c8"};
+
+const std::vector<TypePtr> kSupportedTypes = {
+    BOOLEAN(),
```

Comment:
We should continue to use TypeKinds and add a comment clarifying that the test only covers TypeKind pair.
To build the RowType below, we can use `fromKindToScalerType()` to get TypePtr